moved/src/genesis/framework.rs

```diff
@@ -49,6 +52,16 @@ static SUI_SYSTEM_PACKAGES: Lazy<BTreeMap<ObjectID, SystemPackage>> = Lazy::new(
     snapshot.into_iter().map(|pkg| (pkg.id, pkg)).collect()
 });
 
+const fn eth_address(value: &[u8; 20]) -> AccountAddress {
+    let mut buf = [0u8; 32];
+    let mut i = 0;
+    while i < 20 {
```

Comment:
nit: `for i in 0..20 {` might be easier to read

Reply:
`for` loops are currently not allowed in `const` functions unfortunately.